int16 targetBody
---
bool status
float64[] targetBodyPose

